website/source/api/system/leases.html.md

```diff
@@ -90,7 +90,7 @@ $ curl \
 
 ## Renew Lease
 
-This endpoint renews a lease, requesting to extend the lease.
+This endpoint renews a lease, requesting to extend the lease. Be sure to check the returned TTL / lease duration to make sure that it has actually increased, as you may be limited by max TTL. A renewal request can still return a 200 OK even if the lease duration has not increased.  
```

Comment:
Rather than retread this info here I feel like it'd be better to point to https://www.vaultproject.io/docs/concepts/lease.html#lease-durations-and-renewal so that if anything changes we can ensure it's updated in both locations.